src/plugins/api/qmlpluginapi.h

```diff
@@ -81,19 +81,21 @@ class PluginAPI : public mu::engraving::QmlPlugin
     INJECT(plugins, mu::context::IGlobalContext, context)
 
     /** Path where the plugin is placed in menu */
-    Q_PROPERTY(QString menuPath READ menuPath WRITE setMenuPath)
```

Comment:
I would recommend not changing the old API. It definitely requires some redesing, but that's not for this PR